### ISSUE TITLE
SCT not showing

### ISSUE THREAD

**Issue by Ashe-H** [Artenuvielle/GW2-SCT#53]
Hello, 

I'm new to GW2 coming from wow and since we have SCT in wow I kinda wanted to have a similar experience so I decided to install GW2-SCT but for some reason it made my game crash sometimes but the main issue is its not showing at all.

When I go to the Scroll Areas tab and try to edit the location of any text it doesn't work, its stuck at 0 and the 00.00 are shaking so hard when I drag it it shows a different numbers but it goes back to 0 again.

Anyone experiencing the same issue or have an idea about why SCT isn't showing and why I can't even edit stuffs ?

Thanks in advance

**Comment by shrugal** [comment]
Same here. I can open and browser the settings menu, but nothing else shows up on screen, and scroll area offsets and dimensions reset to 0.000 immediately.

Fresh install using the Addon-Manager, and running on DX11.

**Comment by TanekDK** [comment]
I have the same issue, and I tried letting UOAOM manage the addons, as well as downloading arcdps and arcdps SCT from their respective official sites. The former method snaps the scroll area parameters to 0.000, and shows no boxes. The latter method shows the default boxes, but as soon as I start doing damage to something, arcdps instantly disables itself with a warning, that behavior is unpredictable.

**Comment by thecynicalpaul** [comment]
Yeah, it's pretty much unusable right now on DX11. I get immediate crashes when engaging in combat that would otherwise trigger the scrolling text. Tried different combinations of dlls and different ways to run it (through UOAOM and stand-alone). 

The jittering between 0 and 1 for scroll areas is caused by trying to run the pre-v2.0 versions of SCT with DX11 enabled. This happens with UOAOM, because it has outdated version for SCT.

**Comment by Grapjas93** [comment]
Can confirm this mod is still working with the newest release on Dx11

**Comment by Sereath** [comment]
Sometimes it works (The first day I started using it went great). And sometimes the game just crashes when entering combat (a day right after).
Unfortunate.

https://gist.github.com/Sereath/bc6943376340b56cb6476c68aca9a57f
